site/content/docs/0.4/components/badge.md

```diff
@@ -1,11 +1,120 @@
 ---
 layout: docs
 title: Badges
```

Comment:
It might come from Bootstrap, but the title isn't the same as the one displayed in the sidebar, do we correct it ?
Side question, in the sidebar, to harmonize, do we use plural or singular for the components ? (Badge vs Badges)

Reply:
Let's bring the point to other team member